setup.py

```diff
@@ -28,24 +28,23 @@ def findfiles(pat):
             ],
         },
     install_requires=[
-        'path.py',
-        'argparse',
```

Comment:
why did you remove this ?

Reply:
argparse is part to the Standard library. @andrey-canon